htdocs/core/class/stats.class.php

```diff
@@ -86,7 +86,7 @@ public function getNbByMonthWithPrevYear($endyear, $startyear, $cachedelay = 0,
 				$year = $year - 1;
 			}
 			while ($year <= $endyear) {
-				$datay[$year] = $this->getNbByMonth($year, $format);
+				$datay[$year] = $this->_getNbByMonth($year, $format);
```

Comment:
The method getNbByMonthWithPrevYear is never used by an instance. This class is abtracts. There is another class that inherit it and that is used to call getNbByMonthWithPrevYear().
And the getNbByMonth() is a method of this parent class.
So i think current code is good.